ote/resources/public/language/en.edn

```diff
@@ -1349,6 +1352,38 @@ You can also draw the operating area or point to the map with drawing tools. You
   :check-email-for-link "Your request for changing password has been received. A link for resetting your password has now been sent to your email address."
   :logged-out "You have been logged out."}
 
+ :service-viewer
+ {:open-in-geojson "Avaa tiedot GeoJSON-muodossa"
```

Comment:
Most of these or even maybe all are translated in viewer section above

Reply:
They were slightly differently worded so I thought it was just safer to add new ones.